commands/src/main/java/org/wildfly/extras/creaper/commands/orb/OnOff.java

```diff
@@ -0,0 +1,18 @@
+package org.wildfly.extras.creaper.commands.orb;
+
+public enum OnOff {
```

Comment:
This doesn't seem to be exposed anywhere, so it shouldn't be `public`. Other enums seem to be used, so it's just this one.